src/components/MoneyRequestHeader.tsx

```diff
@@ -79,16 +79,11 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
     const isScanning = TransactionUtils.hasReceipt(transaction) && TransactionUtils.isReceiptBeingScanned(transaction);
     const isPending = TransactionUtils.isExpensifyCardTransaction(transaction) && TransactionUtils.isPending(transaction);
 
-    const isRequestModifiable = !isSettled && !isApproved && !ReportActionsUtils.isDeletedAction(parentReportAction);
```

Comment:
I've removed this as it's confusing while we already have `canModifyRequest`. Also, this is only used to decide if user can hold or unhold the request, so I renamed it accordingly.